src/core_functions/aggregate/holistic/quantile.cpp

```diff
@@ -507,15 +512,75 @@ struct QuantileBindData : public FunctionData {
 		deserializer.ReadProperty(100, "quantiles", raw);
 		deserializer.ReadProperty(101, "order", result->order);
 		deserializer.ReadProperty(102, "desc", result->desc);
+		int deserialization_kind;
+		deserializer.ReadPropertyWithDefault(103, "flag", deserialization_kind, 0);
+
+		if (deserialization_kind) {
+			LogicalType type;
+			deserializer.ReadProperty(104, "logical_type", type);
+
+			switch (deserialization_kind) {
```

Comment:
My only comment is that I think this would be cleaner and more consistent with the rest of the code if it were turned into an inheritance chain. You could then use the serialization generation script to generate this code.

Also turning the "flag" into "type" enum. Remember to run the `generate_enum_util` script too.